de.haeckerfelix.gradio.json

```diff
@@ -0,0 +1,135 @@
+{
+    "app-id": "de.haeckerfelix.gradio",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "gradio",
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--device=dri",
+        "--talk-name=org.freedesktop.Tracker1",
+        "--socket=pulseaudio",
+        "--talk-name=org.gtk.vfs", "--talk-name=org.gtk.vfs.*",
+        "--talk-name=org.gnome.ScreenSaver",
+        "--talk-name=org.gnome.SettingsDaemon",
+        "--talk-name=org.mpris.MediaPlayer2",
+        "--talk-name=org.mpris.MediaPlayer2.Player",
+        "--own-name=org.mpris.MediaPlayer2.gradio",
+        "--talk-name=org.gnome.SettingsDaemon",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.gnome.SessionManager",
+	"--filesystem=xdg-music:ro"
+    ],
+    "build-options" : {
+        "cflags": "-O0 -g",
+        "cxxflags": "-O0 -g",
+        "env": {
+            "GST_PLUGIN_SYSTEM_PATH": "/app/lib/gstreamer-1.0/"
+        }
+    },
+    "modules": [
+        {
+            "name": "gstreamer",
```

Comment:
gstreamer is already in the runtime, is there anything you are missing from it?

Reply:
Where i can see which codecs are included in the runtime? Or where can I generally see which software is included?

Reply:
You can look here: https://github.com/GNOME/gnome-sdk-images/tree/gnome-3-24

Or `flatpak run org.gnome.Sdk//3.24` and dig around the filesystem yourself.